packages/contracts/package.json

```diff
@@ -19,11 +19,11 @@
     "publish:npm": "yarn prepublish && npm publish"
   },
   "devDependencies": {
+    "@nomiclabs/hardhat-etherscan": "^3.1.0",
     "@primitivefi/hardhat-dodoc": "0.2.3",
     "@unlock-protocol/hardhat-helpers": "workspace:^",
     "@unlock-protocol/tsconfig": "workspace:./packages/tsconfig",
     "@unlock-protocol/types": "workspace:^",
-    "@nomiclabs/hardhat-etherscan": "^3.1.0",
```

Comment:
Been wondering why this dependency keep changing place in the hierarchy 🤔

Reply:
yarn makes deps follow alphabetical order - I added it manually. Which makes me think I need to remove that caret in the version.